app.yaml

```diff
@@ -143,3 +143,15 @@ libraries:
 # running the server in the dev environment.
 env_variables:
   MINIFICATION: false
+
+
+skip_files:
+# .pyc and .pyo files
+- ^(.*/)?.*\.py[co]$
+# Unix hidden files whose names begin with a dot
+- ^(.*/)?\..*$
+# Karma test files
+- ^(.*/)Spec.js$
+# Other folders to ignore
+- tests/
```

Comment:
Note: I can't omit data/, since the dev server actually needs it. The dev server obeys this skip_files attribute.